examples/relay/multiplayerRelay.js

```diff
@@ -0,0 +1,32 @@
+const { Relay } = require('bedrock-protocol')
```

Comment:
Don't need another example, this can be removed

Reply:
Okay